apps/datahub/src/app/dataset-view/record-view.component.ts

```diff
@@ -9,6 +10,10 @@ import { MdViewFacade } from '@geonetwork-ui/feature/search'
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RecordViewComponent {
+  displayMap$ = this.facade.mapApiLinks$.pipe(
+    map((links) => !!links && links.length > 0)
```

Comment:
this observable should not emit anything other than arrays... this may be why also there are sometimes errors when opening a record

Reply:
but this can be addressed in another PR I guess